modules-pr-backporter/library_multi_pull_requests.py

```diff
@@ -18,6 +18,7 @@
 import os
 import subprocess
 import sys
+import wget
```

Comment:
Where does the `wget` module come from?

Reply:
https://pypi.org/project/wget/
I think this will require an explicit pip install. I still haven't tested it on a Github machine.

Reply:
Done.

Reply:
I don't understand why we would want to use the `wget` module? We can just use `urllib3`?

Reply:
I think `wget` is simpler, but it seems that `urllib3` is safer so I went ahead and switched to `urllib3`.

Reply:
Only using external modules if they provide lots of value, otherwise prefer standard library packages.